src/Mod/Draft/DraftEdit.py

```diff
@@ -57,9 +58,13 @@
 
 
 class Edit():
-
-    "The Draft_Edit FreeCAD command definition"
-
+    """
+    The Draft_Edit FreeCAD command definition.
+    A tool to graphically edit FreeCAD objects.
+    Current implementation use many parts of pivy graphics code by user "looo".
+    The tool collect editpoints from objects and display Trackers on them to allow
+    editing their Shape and their parameters.
+    """
```

Comment:
I see that you added a lot of documentation strings. That's good.

For the class, you can add a short explanation of the attributes, that is, all those variables like `self.running`, `self.trackers`, etc., without the word `self.`

```py
Class Edit():
    """
    Attributes
    ------
    running : bool
        Whether the tool is running or not.
    trackers : dictionary
        The dictionary that holds the current object and its tracker.
    ...
    """
```

Reply:
Yes! i'll do it

Reply:
Verbosity is not a problem. As a matter of fact we want the docstrings to be verbose. Then we don't have to write separate documentation, we can just take a look at the source. Tools like Doxygen and Sphinx parse these documentation strings and produce good documentation that can be used by both users and programmers.

Take a look at WorkingPlane.py to get an idea how I wrote the documentation strings there.